Airship.iOS.Basement.nuspec

```diff
@@ -15,7 +15,7 @@
 
    <files>
      <file src="src/AirshipBindings.iOS.Basement/bin/release/AirshipBindings.iOS.Basement.dll" target="lib/Xamarin.iOS10" />
-     <file src="src/AirshipBindings.iOS.Basement/bin/Release/AirshipBindings.iOS.Basement.resources/**" target="content/UABasement.resources" />
-     <file src="src/AirshipBindings.iOS.Basement/urbanairship.ios.basement.targets" target="build/urbanairship.ios.basement.targets" />
+     <file src="src/AirshipBindings.iOS.Basement/bin/Release/AirshipBindings.iOS.Basement.resources/**" target="content/ABasement.resources" />
```

Comment:
This seems to be inside our module right? Can we just say `content/res.resources` ?